index.html

```diff
@@ -262,21 +287,21 @@ <h3>VerifiableCredentialSchema2023</h3>
                    </tr>
                    <tr>
                       <td>type</td>
-                      <td>The <code>type</code> property MUST be VerifiableCredentialSchema2023</td>
+                      <td>The <code>type</code> property MUST be JsonSchemaCredential2023</td>
                    </tr>
                 </tbody>
                </table>
             </p>
                An example of utilizing the VC Data Model's <code>credentialSchema</code>
                is provided below:
-               <pre class="example" title="Example VerifiableCredentialSchema2023">
+               <pre class="example" title="Example JsonSchemaCredential2023">
                {
                  "@context": [
                    "https://www.w3.org/ns/credentials/v2",
                    "https://www.w3.org/ns/credentials/examples/v2"
                  ],
                  "id": "https://example.com/credentials/3733",
-                 "type": ["VerifiableCredential", "EmailCredential"],
+                 "type": ["VerifiableCredential", "JsonSchemaCredential2023", "EmailCredential"],
```

Comment:
Is this intentional? I was expecting the top level credential (the one that's declaring the `credentialSchema`), to not have this type.

Reply:
Agree with @andresuribe87 ... the top-level VC is an "EmailCredential" that contains a reference to a JSON Schema in order to validate the syntax of the VC.